src/Uno.Extensions.Reactive.UI.Tests/Presentation/Bindings/Collections/Given_BindableCollection_Selection.cs

```diff
@@ -107,4 +181,16 @@ private async Task<(BindableGiven_BindableCollection_Selection_Model, ListView,
 
 		return (vm, lv, lvItems);
 	}
+
+	private async Task<(BindableGiven_BindableCollection_Selection_Model, ComboBox, MyItem[])> SetupComboBox()
+	{
+		var vm = new BindableGiven_BindableCollection_Selection_Model();
+		var cb = new ComboBox { ItemsSource = vm.Items };
+
+		await UIHelper.Load(cb, CT);
+
+		var items = vm.Items.ToArray();
```

Comment:
This `ToArray()` does not exists, please remove the `items` as you are not using them anyway.